core/src/main/java/org/frankframework/monitoring/MonitorManager.java

```diff
@@ -52,7 +53,7 @@ public class MonitorManager extends AbstractConfigurableLifecyle implements Appl
 
 	private @Getter @Setter ApplicationContext applicationContext;
 	private final List<Monitor> monitors = new ArrayList<>();                            // All monitors managed by this MonitorManager
-	private final Map<String, Event> events = new HashMap<>();                           // All events that can be thrown
+	private final Map<String, Event> events = new ConcurrentHashMap<>();                 // All events that can be thrown
```

Comment:
Since I made the handling of event asynchronous, it turns out this map can now be updated from multiple different threads during startup.
So it needs to be concurrency-safe.